pkg/component/monitoring/prometheus/prometheus.go

```diff
@@ -95,4 +95,17 @@ func (p *prometheus) prometheus() *monitoringv1.Prometheus {
 			RuleNamespaceSelector: &metav1.LabelSelector{},
 		},
 	}
+
+	if takeOverOldPV {
+		obj.Spec.InitContainers = append(obj.Spec.InitContainers, corev1.Container{
+			Name:            "take-over-old-pv",
+			Image:           p.values.DataMigration.ImageAlpine,
+			ImagePullPolicy: corev1.PullIfNotPresent,
+			VolumeMounts:    []corev1.VolumeMount{{Name: "prometheus-db", MountPath: "/prometheus"}},
+			Command:         []string{"/bin/sh", "-c"},
+			Args:            []string{`if [[ -d /prometheus/prometheus- ]]; then mv /prometheus/prometheus- /prometheus/prometheus-db; else echo "rename already done"; fi`},
```

Comment:
Not sure if it is worth the effort, but do you want to also log something in case the move happens? As it is now, the container either is completely silent or it has "rename already done" as output.